src/sentry/nodestore/django/models.py

```diff
@@ -17,7 +17,7 @@
 class Node(BaseModel):
     __core__ = False
 
-    id = models.CharField(max_length=40, primary_key=True)
+    id = models.CharField(max_length=50, primary_key=True)
```

Comment:
We can't change this without a migration.

Reply:
ah ok. I thought this table was only stood up for tests. but I guess we have actual users of django nodestore?

Reply:
I'd say 99% of them. Since nobody is running Riak.

Reply:
hmm, yes good point